apps/shell/test/allowlist.test.js

```diff
@@ -2,6 +2,7 @@
  * Allowlist helper function tests
  */
 const helpers = require('../utils/helpers')
+const { HostAllowlist } = require('../utils/helpers')
```

Comment:
```js
const helpers = require('../utils/helpers')

const allowlist = new helpers.HostAllowlist()
```

You can access `HostAllowlist` with `helpers.HostAllowlist` instead but `HostAllowlist` should be in its own file instead of in the helper methods now that its a class.